src/ext/dispatch.c

```diff
@@ -457,19 +465,21 @@ int default_dispatch(zend_execute_data *execute_data TSRMLS_DC) {
 }
 
 int ddtrace_wrap_fcall(zend_execute_data *execute_data TSRMLS_DC) {
-    const char *function_name = NULL;
-    uint32_t function_name_length = 0;
     DD_PRINTF("OPCODE: %s", zend_get_opcode_name(EX(opline)->opcode));
+    if (DDTRACE_G(disable) || DDTRACE_G(disable_in_current_request)) {
+        return default_dispatch(execute_data);
```

Comment:
One little ZTS thing. :)

```suggestion
        return default_dispatch(execute_data TSRMLS_CC);
```

Reply:
RIght, I'll also add compilation test to make sure this is caught by CI.